src/main/java/pl/gispartner/ResourceManagerApp/service/ResourceService.java

```diff
@@ -0,0 +1,22 @@
+package pl.gispartner.ResourceManagerApp.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import pl.gispartner.ResourceManagerApp.model.ResourceEntity;
+import pl.gispartner.ResourceManagerApp.persistance.ResourceRepository;
+
+@Service
+@RequiredArgsConstructor
+public class ResourceService {
+
+    private final ResourceRepository resourceRepository;
+
+    public void saveResource(ResourceEntity resourceEntity) {
```

Comment:
przerob to tak aby nie przekazywac z frontu ResourceEntity tylko np. ResourceRequest ktory nastepnie przemapujesz na ResourceEntity (przez builder)